src/lib.rs

```diff
@@ -123,6 +123,9 @@ mod init;
 pub mod pbkdf2;
 
 pub mod rand;
+mod limb;
+pub use limb::GFp_rand_mod;
```

Comment:
My understanding was that this is necessary to make it visible to the C code. Without this, I get:

```error: function GFp_rand_mod is marked #[no_mangle], but not exported```

Is there another way around that?